### PR TITLE
clean code

### PR DESCRIPTION
**Pull request by divyankgarg**
Review clean code

### CODE REVIEW

File: data_ingestion.py [hunk 1]
```diff
@@ -25,23 +25,27 @@ def get_emission_data(self):
...
-				pos = re.search(r'\,.*', City_name).start() 
-				return City_name[:pos] 
-			else:
-				return City_name
-		df_emission_data['city'] = df_emission_data['city'].apply(Clean_names) 
+		df_emission_data = pd.read_json(emission_data) 
 		print(df_emission_data.head())
 		return df_emission_data
 
+	def Clean_names(self):
```
Comment (biharicoder) [inline data_ingestion.py]: Function name should be lower case:
https://www.python.org/dev/peps/pep-0008/#id45

File: data_ingestion.py [hunk 2]
```diff
@@ -25,23 +25,27 @@ def get_emission_data(self):
...
+		"""To remove the state name from city column
+		Parameters: City_name - to look upon city column
+		Returns: A dataframe where state short form removed from city column
+		"""
+		k=[]
```
Comment (biharicoder) [inline data_ingestion.py]: variable names should be self-explanatory

File: data_ingestion.py [hunk 3]
```diff
@@ -25,23 +25,27 @@ def get_emission_data(self):
...
+		df_emission_data=self.get_emission_data()
+		for x in df_emission_data.city:
```
Comment (biharicoder) [inline data_ingestion.py:40]: Use lamda function and apply method instead of looping when using pandas dataFrame

File: data_ingestion.py (hunk 1 above)
Comment (divyankgarg) [reply data_ingestion.py]: done

File: data_ingestion.py (hunk 2 above)
Comment (divyankgarg) [reply data_ingestion.py]: done

**Pull request closed without merging**